src/realtime/tasks/tests/test_celery_tasks.py

```diff
@@ -0,0 +1,175 @@
+# coding=utf-8
+import json
+import os
+import shutil
+import unittest
+from threading import Thread
+
+import mock
+import pytz
+from http.server import HTTPServer
+
+from realtime import settings
+from realtime.tasks.earthquake import process_shake
+from realtime.tasks.flood import process_flood
+from realtime.tasks.generic import check_broker_connection
+from realtime.tests.mock_server import InaSAFEDjangoMockServerHandler
+
+
+class TestGenericCeleryTasks(unittest.TestCase):
+
+    def test_broker_connection(self):
+        """Test broker connection check."""
+
+        result = check_broker_connection.delay()
+
+        self.assertTrue(result.get())
```

Comment:
@ismailsunni try to make something simple like this running using pycharm unittests